README.md

```diff
@@ -17,6 +17,11 @@ send them timely push notifications via [FCM (Firebase Cloud Messaging)](https:/
 > ⚠️ **We support Android API level 23 and above** ⚠️
 
 ## Installation
+
+### Requirements
+
+- Kotlin **1.9.0** or later
```

Comment:
🔍(nit pick)  Should this be like a info bar instead? Now it seems like the below section which formerly was installation is now requirements.

Reply:
I kinda felt like this would be a sub-category for installation, but I can make it into a new section if you prefer. Where would you like to see it?

Reply:
theres an info block above this actually that about the Android API version, maybe we can join those together? probably also make the wording for the Android API match this

Reply:
Sorry I wasn't clear. Kenny is right, that is what I meant.